server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -665,6 +666,14 @@ func (s *BuildBuddyServer) GetExecution(ctx context.Context, req *espb.GetExecut
 	return nil, status.UnimplementedError("Not implemented")
 }
 
+func (s *BuildBuddyServer) GetExecutionNode(ctx context.Context, req *scpb.GetExecutionNodeRequest) (*scpb.GetExecutionNodeResponse, error) {
+	if ss := s.env.GetSchedulerService(); ss != nil {
+		res, err := ss.GetExecutionNode(ctx, req)
```

Comment:
nit: you could also write `return ss.GetExecutionNode(ctx, req)` to directly return the response and error values